### ISSUE TITLE
Fix TikTok

### ISSUE THREAD

**Issue by Erk-** [Erk-/rsget#4]

**Comment by dvtkrlbs** [comment]
I think we can just get the vide webpage and extract the video src.

**Comment by Erk-** [comment]
Yeah that would be possible, but other things we would like to extract is
the author name and the title of the clip

Den ons. 24. apr. 2019 kl. 10.12 skrev Tunahan Karlıbaş <
notifications@github.com>:
